float64 traveling_time
float64 distance
RoadInfo[] roadinfo
